src/redux/slices/support/selector.ts

```diff
@@ -3,8 +3,21 @@ import { createSelector } from '@reduxjs/toolkit'
 
 const selector = (state: RootState) => state.support
 
-export const support = createSelector([selector], (state) => state.support)
+export const help = createSelector([selector], (state) => state.support.help)
+export const isHelpMessageSuccess = createSelector(
+  [selector],
+  (state) => state.support.help.isHelpMessageSuccess
+)
+export const contactUs = createSelector([selector], (state) => state.support.contactUs)
+
+export const isContactUsMessageSuccess = createSelector(
+  [selector],
+  (state) => state.support.contactUs.isContactUsMessageSuccess
+)
```

Comment:
change to isContactUsDataSubmittedSuccess

Reply:
done